src/widgets/ot_color_picker.eliomi

```diff
@@ -0,0 +1,70 @@
+[%%shared.start]
+
+(** Ot_color_picker : this module allows to generate, client side, a color
+    selector. This selector consists of a color table and an html div that
+    displays the current selected color. *)
+
+(** The main type of Ot_color_picker module *)
+type t
+type div = Html_types.div Eliom_content.Html.D.elt
+
+(** The argument is the divisor of 255. It have to be greater than 1 *)
+val generate_color_samples : int -> string list list list
+
+(* Some pre-generated color samples in several precision. Color samples
+   are list of list of list of colors represented in string of hexadecimal
+   values.*)
```

Comment:
We need `(**` for this to appear in the doc.

You should probably leave a space before `color_samples_p2` so that the comment refers to all the `color_samples_p*` that follow.